data/yam/agama/auto/agama_auto_tumbleweed.yaml

```diff
@@ -0,0 +1,3 @@
+---
+test_data:
+  product: openSUSE Tumbleweed
```

Comment:
we might change the way to validate the product, the hash key should be:
`os_release_name` which is what you compare later, it might be very subtle ...but in my experience it is a mess with test data if not named properly and sometimes we have found almost similar values for similar things, like imaging there is a name and name_blabla, which is the same but lower letter and you also want to compare it, so then using exact name of what you compare in test data is faster to understand the test.